test/sql/bind.result

```diff
@@ -307,7 +307,7 @@ test_run:cmd("setopt delimiter ';'")
 if remote then
 	execute("CREATE TABLE t(a VARBINARY PRIMARY KEY);")
 	execute("INSERT INTO t VALUES (X'00');")
-	res = execute("SELECT typeof(?);", box.space.T:select()[1])
+    res = execute("SELECT TYPEOF(?);", box.space.t:select()[1])
```

Comment:
Typo: misindent (will be fixed by result file update).

Reply:
See above.